From f5aea82d0d9d3d181a98e4f8171f483c51aab429 Mon Sep 17 00:00:00 2001
From: John Linn <john.linn@xilinx.com>
Date: Sun, 8 Apr 2012 22:01:18 -0700
Subject: [PATCH] Xilinx: I2C: Remove compile warning from Si570 driver

Just minor cleanup to remove a compiler warning, removing
an unused variable.
---
 drivers/misc/si570.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/misc/si570.c b/drivers/misc/si570.c
index 031289d..ffac0df 100644
--- a/drivers/misc/si570.c
+++ b/drivers/misc/si570.c
@@ -209,7 +209,6 @@ static int si570_set_frequency(struct i2c_client *client,
 {
 	int i, n1, hs_div;
 	u64 fdco, best_fdco = ULLONG_MAX;
-	u8 dco_reg;
 
 	for (i = 0; i < ARRAY_SIZE(si570_hs_div_values); i++) {
 		hs_div = si570_hs_div_values[i];
-- 
1.7.0.4

